source/contracts/Augur.sol

```diff
@@ -3,13 +3,98 @@ pragma solidity 0.4.17;
 
 import 'Controlled.sol';
 import 'libraries/token/ERC20.sol';
+import 'reporting/IUniverse.sol';
 
 
 // AUDIT/CONSIDER: Is it better that this contract provide generic functions that are limited to whitelisted callers or for it to have many specific functions which have more limited and specific validation?
 contract Augur is Controlled {
+    event MarketCreated(address indexed universe, address indexed market, address indexed marketCreator, uint256 marketCreationFee, string extraInfo);
+    event DesignatedReportSubmitted(address indexed universe, address indexed reporter, address indexed market, address stakeToken, uint256 amountStaked, uint256[] payoutNumerators);
+    event ReportSubmitted(address indexed universe, address indexed reporter, address indexed market, address stakeToken, uint256 amountStaked, uint256[] payoutNumerators);
+    event WinningTokensRedeemed(address indexed universe, address indexed reporter, address indexed market, address stakeToken, uint256 amountRedeemed, uint256 reportingFeesReceived, uint256[] payoutNumerators);
+    event ReportsDisputed(address indexed universe, address indexed disputer, address indexed market, uint8 reportingPhase, uint256 disputeBondAmount);
+    event MarketFinalized(address indexed universe, address indexed market);
+    event UniverseForked(address indexed universe);
+    event OrderCanceled(address indexed universe, address indexed shareToken, address indexed sender, bytes32 orderId, uint8 orderType, uint256 tokenRefund, uint256 sharesRefund);
+    event OrderCreated(address indexed universe, address indexed shareToken, address indexed creator, bytes32 orderId, uint256 price, uint256 amount, uint256 numTokensEscrowed, uint256 numSharesEscrowed, uint256 tradeGroupId);
+    event OrderFilled(address indexed universe, address indexed shareToken, address indexed creator, address filler, uint256 price, uint256 numCreatorShares, uint256 numCreatorTokens, uint256 numFillerShares, uint256 numFillerTokens, uint256 settlementFees, uint256 tradeGroupId);
+    event ProceedsClaimed(address indexed universe, address indexed sender, address indexed market, uint256 numShares, uint256 numPayoutTokens, uint256 finalTokenBalance);
+    event UniverseCreated(address indexed parentUniverse, address indexed childUniverse);
```

Comment:
I don't see any transfer events included in here.  @tinybike @epheph, this means that any accounting you do in the DB won't line up.  For example, if account 0xabcd buys stake tokens then transfers them to 0x1234 who redeems them, you will end up with a redeem that has no correlated buy.  The same can happen for shares and any of the other tokens, which means when the UI attempts to show trading history, P/L calculations, transaction history, etc. the balance sheets won't add up.

Reply:
The Transfer event is in our `StandardToken` and `BasicToken` implementations

Reply:
Yes, but for the same reason we centralized the rest of the events in Augur.sol, we also need to centralize those transfer events.  This does mean that the Transfer event will be fired twice, once for ERC20 purposes, and once for Augur Node purposes.

Reply:
I previously mentioned to @nuevoalex that it was ok to leave the proposed `TokensTransfered` event out, but thinking about it more, I think @MicahZoltu is right about this -- the extra gas cost of logging twice (once for ERC20, once for Augur) is IMO outweighed by the convenience of having all the relevant logs accessible from a single contract.  Could we add `TokensTransfered` to Augur.sol?

Reply:
Yea. We don't actually have to log twice either really. To implement this I was imagining giving the base token a method to emit logs which will get called during transfers, where it emits what it currently does but subclasses can override however they choose (In our case sending a request to Augur to log). Do we _want_ 2 logs?

Reply:
We need 2 logs if we want our tokens to be ERC20 tokens.  For ReputationToken at least, we do want this and presumably for others as well.